integration/kubernetes/confidential/fixtures/pod-config-secret.yaml.in

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+apiVersion: v1
+kind: Pod
+metadata:
+  name: secret-test-pod-cc
+spec:
+  runtimeClassName: $RUNTIMECLASS
+  containers:
+  - name: busybox
+    image: quay.io/prometheus/busybox:latest
+    imagePullPolicy: Always
+    command:
+    - sh
+    - -c
+    - |
+      env
+      if env | grep -q "SECRET_PASSWORD=unsealed_secret"; then
+        echo "unsealed environment as expected"
```

Comment:
Maybe printing the value of `$SECRET_PASSWORD` in this message might be helpful for debug?

Reply:
Yes, I use `$SECRET_PASSWORD` and found replace to null by `envsubst`, I changed to only substitute `RUNTIMECLASS` and switch back to use `$SECRET_PASSWORD` now.